templates/base.html

```diff
@@ -0,0 +1,181 @@
+                                    data-parent="#sidenavAccordionPages">
+                                    <nav class="sb-sidenav-menu-nested nav">
+                                        <a class="nav-link" href="/historical/enrolled_grade">National Enrollment /
+                                            Grade</a>
+                                        <a class="nav-link" href="/historical/district_grade">District Enrollment /
+                                            Grade</a>
+                                        <a class="nav-link" href="/historical/#">Enrollment / Grade by School Trend</a>
+                                        <a class="nav-link" href="/historical/#">Enrollment / Grade by District
+                                            Trend</a>
+                                        <a class="nav-link" href="/historical/#">Enrollment Summary</a>
+                                    </nav>
+                                </div>
+                            </nav>
+                        </div>
+                    </div>
+                    {% endif %}
+                    {% if 'auth' in request.path %}
+                    <div class="nav">
+                        <div class="sb-sidenav-menu-heading">System Admin</div>
+                        <a class="nav-link" href="{% url 'authentication:user-directory' %}">
+                            <div class="sb-nav-link-icon"><i class="fas fa-tachometer-alt"></i></div>
+                            User Directory
+                        </a>
+                        <a class="nav-link" href="{% url 'authentication:create-user' %}">
+                            <div class="sb-nav-link-icon"><i class="fas fa-tachometer-alt"></i></div>
+                            Create User
+                        </a>
+                        <a class="nav-link" href="{% url 'authentication:user-detail' user.id%}">
+                            <div class="sb-nav-link-icon"><i class="fas fa-tachometer-alt"></i></div>
+                            User Profile
+                        </a>
+
+                    </div>
+                    {% endif %}
+                </div>
+                <div class="sb-sidenav-footer">
+                    {% if user.is_authenticated %}
+                    <div class="small">Logged in as: {{user.username}}</div>
+                    {% endif %}
+                </div>
+            </nav>
+        </div>
+        <div id="layoutSidenav_content">
+            <main>
+                <div class="container-fluid">
+                    {% block page_content %}{% endblock %}
+                </div>
+            </main>
+        </div>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js"
+        crossorigin="anonymous"></script>
+    <script src="{% static 'js/scripts.js' %}"></script>
```

Comment:
Can we move these scripts into the head section? I have some scripts on the user detail page that depend on jquery (i.e. setting the search term, navigating the user to the proper user profile page).
http://localhost:8000/auth/sysadmin/users